Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM node:14
 RUN set -x \
   && git clone https://github.com/DataBiosphere/terra-batch-analysis-ui.git \
   && cd terra-batch-analysis-ui \
-  && git checkout main \
+  && git checkout ss_submit_workflow \
```

Comment:
Since this same dockerfile is used for building images that are published to GCR, I had kept it to be checked out from `main`. Plus I am thinking that once [BW-1297](https://broadworkbench.atlassian.net/browse/BW-1297) is done, we might no longer need this for creating images and pushing it to GCR and we can update this file for locally building images to test out the UI.

Reply:
Definitely a topic for another ticket 😄 ... I would expect the "push to GCR" action to look kind of like:

1. Check out main
2. Build Dockerfile on what you just checked-out
3. Push to GCR

Which I don't think would be inconsistent with a local build of

1. Check out (or update) any branch
2. Build Dockerfile on what you just checked-out
3. Push to GCR